Bottom: 163e43048e0e8d45f32c7f7589ffd95f78c54fb9
Top:    145a50cc09a8e694ae7d5f7dbaaac7f5b7a1a89b
Author: aingerson <alexia.ingerson@intel.com>
Date:   2018-11-27 13:00:19 -0800

Change fabtests source

The ofiwg/fabtests repo is frozen and was merged with the libfabric repo


---

diff --git a/single-node.sh b/single-node.sh
index c2f2cf2..1eaef64 100644
--- a/single-node.sh
+++ b/single-node.sh
@@ -29,7 +29,15 @@ make -j 4
 sudo make install
 
 # Runs all the tests in the fabtests suite while only expanding failed cases
+declare EXCLUDE=$WORKSPACE/fabtests/install/share/fabtests/test_configs/$PROVIDER/${PROVIDER}.exclude
+if [ -f $EXCLUDE ]; then
+	EXCLUDE="-R -f $EXCLUDE"
+else
+	EXCLUDE=""
+fi
+
 echo "==> Running fabtests"
 LD_LIBRARY_PATH=$WORKSPACE/fabtests/install/lib/:$LD_LIBRARY_PATH	\
 BIN_PATH=$WORKSPACE/fabtests/install/bin/ FI_LOG_LEVEL=debug		\
-$WORKSPACE/fabtests/install/bin/runfabtests.sh -v $PROVIDER 127.0.0.1 127.0.0.1
+$WORKSPACE/fabtests/install/bin/runfabtests.sh -v $EXCLUDE		\
+$PROVIDER 127.0.0.1 127.0.0.1
